gcloud-java-storage/src/test/java/com/google/gcloud/storage/RemoteStorageHelperTest.java

```diff
@@ -139,7 +139,7 @@ public void testForceDeleteTimeout() throws InterruptedException, ExecutionExcep
     }
     EasyMock.expect(storageMock.delete(BUCKET_NAME)).andThrow(RETRYABLE_EXCEPTION).anyTimes();
     EasyMock.replay(storageMock);
-    assertTrue(!RemoteGcsHelper.forceDelete(storageMock, BUCKET_NAME, 50, TimeUnit.MICROSECONDS));
+    assertTrue(!RemoteStorageHelper.forceDelete(storageMock, BUCKET_NAME, 50, TimeUnit.MICROSECONDS));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [assertTrue(!expr) can be replaced by assertFalse(expr)](https://www.codacy.com/app/mziccard/gcloud-java/file/2329630538/issues/source?bid=3123103&fileBranchId=3205624#l142)